react-native/src/main/scala/hello/world/PlayAudio.scala

```diff
@@ -0,0 +1,53 @@
+package hello.world
+
+import slinky.core.FunctionalComponent
+import slinky.core.annotations.react
+import slinky.native.View
+import typings.antDesignReactNative.components._
+import typings.antDesignReactNative.{antDesignReactNativeStrings => antdStrings}
+import typings.expoAv.aVMod.AVPlaybackStatusToSet
+import typings.expoAv.mod.Audio.Sound
+import typings.expoAv.anon.Headers
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+@react object PlayAudio {
+
+  type Props = Unit
+
+  val component = FunctionalComponent[Props] { _ =>
+    val uri: String = "https://upload.wikimedia.org/wikipedia/commons/b/b2/Funky_bassline_F7-Bb7.ogg"
+    val soundObject = new Sound
+
+    soundObject
+      .loadAsync(
+        Headers(uri = uri),
+        AVPlaybackStatusToSet().setShouldPlay(false) // optional, but you can set a bunch of stuff here
+      )
+      .toFuture
```

Comment:
you only need `toFuture` if you want to do anything with it afterwards

Reply:
you decide how much effort you want to put into it. I think I glanced from the documentation that you're supposed to close these `soundObject`s? You could see how/if that's done elsewhere

Reply:
> you decide how much effort you want to put into it. I think I glanced from the documentation that you're supposed to close these `soundObject`s? You could see how/if that's done elsewhere

Done it I think in the correct way with `useEffect` hook

Reply:
> you only need `toFuture` if you want to do anything with it afterwards

I thought about using `AVPlaybackStatus` to update the enable/disable the relevant buttons, something like this:

|Status|`Play`|`Pause`|`Stop`|comment|
|----|----|-----|-------|-----|
|stopped|▶|Disabled|Disabled|`Stop` disabled when stopped|
|playing|Disabled|⏸|⏹|`Play` disabled while playing|
|paused|▶|Disabled|⏹|`Pause` enabled while playing|

but I am not sure how relevant this would be for the demo goals.

Reply:
I have marked the PR as draft and pushed a new commit https://github.com/ScalablyTyped/SlinkyDemos/pull/32/commits/5963ee1fbf14f63c356cdee7bb4b9b03f1fe4956 highlighting some issues. 
So even if the previous code seems to work fine, there is something I am not mastering enough to feel confident about publishing and adding to the demo.

Cannot find a way to work with `setOnPlaybackStatusUpdate`, see https://docs.expo.io/versions/v38.0.0/sdk/av/#example-usage.

Seems that if I touch the `AVPlaybackStatus` it provides, the soundObject gets unloaded. Plus I have problems in accessing the values contained in `AVPlaybackStatus`.